rpc_jobs/rpc_aio.yml

```diff
@@ -329,19 +329,23 @@
                   ]
                 aio_prepare.prepare()
                 deploy.deploy_sh(environment_vars: environment_vars)
-                kibana.kibana(kibana_branch)
-                holland.holland()
                 if (env.STAGES.contains("Minor Upgrade")) {{
                   deploy.upgrade_minor(environment_vars: environment_vars)
                 }} else if (env.STAGES.contains("Major Upgrade")) {{
                   deploy.upgrade_major(environment_vars: environment_vars)
+                  tempest.tempest()
                   kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
                   holland.holland()
                 }} else if (env.STAGES.contains("Leapfrog Upgrade")) {{
                   deploy.upgrade_leapfrog(environment_vars: environment_vars)
+                  tempest.tempest()
                   kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+                }} else {{
+                  // in a non-upgrade situation, these tests are run
+                  tempest.tempest()
+                  kibana.kibana(kibana_branch)
+                  holland.holland()
```

Comment:
Tempest and holland are always run, so they can go after the if, then create a new if block to  only run holland if Upgrade is not in stages.

Reply:
Holland is run in the Major upgrade though.

Reply:
That sounds more complex than it should be...

I like it the way it is. However a few tests are missing:
- Minor upgrade is missing kibana and holland
- Leapfrog is missing holland, but that's ok.

Reply:
Leapfrog only ever ran Holland before, not after.